src/middlewared/middlewared/plugins/system.py

```diff
@@ -609,27 +611,29 @@ def _get_index(country_columns, column):
         ip4_addresses_list = [alias_dict['address'] for alias_dict in ip_addresses if alias_dict['type'] == 'INET']
         ip6_addresses_list = [alias_dict['address'] for alias_dict in ip_addresses if alias_dict['type'] == 'INET6']
 
-        ip4_address = data.get('ui_address')
-        if (
-            ip4_address and
-            ip4_address != '0.0.0.0' and
-            ip4_address not in ip4_addresses_list
-        ):
-            verrors.add(
-                f'{schema}.ui_address',
-                'Selected ipv4 address is not associated with this machine'
-            )
+        ip4_addresses = data.get('ui_address')
```

Comment:
What if the user selects wildcard address and another one? Wouldnt that cause problems and/or be ambiguous? Sounds like we should restrict to 0.0.0.0 only if thats selected. Same for IPv6 and ::